examples/pylab_examples/mri_demo.py

```diff
@@ -9,8 +9,7 @@
 
 # Data are 256x256 16 bit integers
 dfile = cbook.get_sample_data('s1045.ima.gz')
-im = np.fromstring(dfile.read(), np.uint16).astype(float)
-im.shape = (256, 256)
+im = np.fromstring(dfile.read(), np.uint16).reshape((256, 256))
```

Comment:
Actually, fromfile doesn't work with gzip.open'ed files (it seems to try to construct an array from the compressed representation).  Reverting to `read()`.